libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/internal/route/line/MapboxRouteLineUtils.kt

```diff
@@ -843,4 +841,11 @@ object MapboxRouteLineUtils {
             else -> y2
         }
     }
+
+    private val decodeRoute: (DirectionsRoute) -> LineString = { route: DirectionsRoute ->
+        LineString.fromPolyline(
+            route.geometry() ?: "",
+            Constants.PRECISION_6
```

Comment:
The route can also be encoded with `PRECISION_5` - https://github.com/mapbox/mapbox-navigation-android/issues/4231.